tests/server/blueprints/cases/test_matchmaker_controllers.py

```diff
@@ -0,0 +1,187 @@
+from http.server import BaseHTTPRequestHandler, HTTPServer
+import socket
+from threading import Thread
+import requests
+import pymongo
+
+from scout.server.blueprints.cases import controllers
+from scout.parse.matchmaker import genomic_features, omim_terms, hpo_terms
+
+MME_ACCEPTS = 'application/vnd.ga4gh.matchmaker.v1.0+json'
+MME_TOKEN = 'test_token'
+
+# mock matchmaker server to send responses when it get requests
+class MockServerRequestHandler(BaseHTTPRequestHandler):
```

Comment:
Again, wow! That is beyond due diligence - really solid!

Reply:
Right? I feel like I'm ready for the pension now :)

Reply:
👵 👍